scripts/west_commands/boards.py

```diff
@@ -106,7 +107,8 @@ def do_run(self, args, _):
                                         revisions=revisions_list,
                                         dir=board.dir, hwm=board.hwm, qualifiers=''))
 
-        for board in list_boards.find_v2_boards(args).values():
+        for board in sorted(list_boards.find_v2_boards(args).values(),
```

Comment:
A "simple" alternative could be to return a sorted list in `find_v2_boards`, similar to `find_boards`.